nav2_waypoint_follower/include/nav2_waypoint_follower/waypoint_follower.hpp

```diff
@@ -141,6 +141,9 @@ class WaypointFollower : public nav2_util::LifecycleNode
   rclcpp::executors::SingleThreadedExecutor callback_group_executor_;
   std::shared_future<rclcpp_action::ClientGoalHandle<ClientT>::SharedPtr> future_goal_handle_;
 
+  // Clock
+  rclcpp::Clock steady_clock_{RCL_STEADY_TIME};
```

Comment:
Remove this clock and use `this->now()` for the node's clock which will use simulation or real time when they are different

Reply:
> Yeah, the extra feedback status needs to be removed. Why is there a larger space between the time taken and the recovery counter? I'd reduce the displayed time elapsed for waypoint following to 1 decimal point. I don't think anyone needs the millisecond level accuracy in this tool

I'm no UI expert, but I believe it's because there are two tables. Changing it to a single table I think would require some more extensive modifications to the formatters and data structures. I'll poke around and see what I can do to fix it. 

 Can change to 1 decimal.